libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -570,13 +629,34 @@ export class CorePivotTableAgImpl extends React.Component<ICorePivotTableProps,
             this.props.onExportReady?.(this.internal.table.createExportFunction(this.props.exportTitle));
         }
 
+        this.updateStickyRow();
         this.updateDesiredHeight();
     };
 
+    /**
+     * This will be called when user changes sorts or adds totals. This means complete re-execution with
+     * new sorts or totals. Loading indicators will be shown instead of all rendered rows thanks to the
+     * LoadingRenderer used in all cells of the left-most column.
+     *
+     * The table must take care to remove the sticky (top-pinned) row - it is treated differently by
+     * ag-grid and will be literally sticking there on its own with the loading indicators.
+     *
+     * Once transformation finishes - indicated by call to onPageLoaded, table can re-instance the sticky row.
+     *
+     * @param _newExecution
+     */
+    private onExecutionTransformed = (_newExecution: IPreparedExecution): void => {
```

Comment:
Do we need the parameter anymore?

Reply:
nope, we don't. i thought keeping it here as unused may be clearer IF we need it in the future.

no strong opinions i can remove it.

Reply:
Fair, leave it in :)